lib/common/client/middleware/response/scorecard_api_errors.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# rubocop:disable Style/StringLiterals
```

Comment:
why?

Reply:
removed here and in spec as well, 404 answer not found